### ISSUE TITLE
Version 2.1.3 Requires Config File

### ISSUE THREAD

**Issue by smlync** [Byaltek/AzureBundling#3]
After updating to 2.1.3, an exception is thrown by GZipBundleConfig, "One or more necessary Configuration settings are missing." We set the properties on Config in the BundleConfig file rather than using the config files. Has the option to not use a config file been removed? The code is below, the exception occurs on the last line. I've also tried setting the BundleCacheTTL, CachePollTime, and UseCompression properties on Config in addition to those in the snippet, but it didn't make a difference.

``` cs
Config.AzureAccessKey = AzureStorageCredentials.AzureAccessKey;
Config.AzureAccountName = AzureStorageCredentials.AzureAccountName;
Config.CdnPath = CdnUrl;
Config.SecureCdnPath = CdnUrl;
BundleTable.VirtualPathProvider = new StorageVirtualPath(AzureContainer, AzureStorageCredentials.AzureAccountName, AzureStorageCredentials.AzureAccessKey);
GZipBundleConfig defaultJSConfig = new GZipBundleConfig("", AzureContainer, 86400, 180, false);
```

Exception Stack Trace:
[ArgumentNullException: Value cannot be null.
Parameter name: One or more necessary Configuration settings are missing.]
   Byaltek.Azure.Config.LoadConfig() +1462
   Byaltek.Azure.GZipBundleConfig.LoadConfig(GZipBundleConfig clone) +88
   Byaltek.Azure.GZipBundleConfig..ctor(String virtualPath, String container, Int32 bundleCacheTTL, Int32 cachePollTime, Nullable`1 useCompression) +143
   solution.com.BundleConfig.RegisterBundles(BundleCollection bundles)...

**Comment by Byaltek** [comment]
The latest version 2.1.8 fixes this issue. It is on nuget.org.

**Comment by smlync** [comment]
Confirmed fixed with version 2.1.8.
